docs/user-quick-start.md

```diff
@@ -99,32 +99,39 @@ enough resources:
 
    Tested with version v1.31.
 
-   Verify you can create a Kubernetes cluster with minikube.
+1. Validate the installation
 
-   ```
-   minikube start -p testcluster
-   ```
-
-   Wait for `testcluster` to complete and then issue this command:
+   Run the drenv-selftest to validate that we can create a test cluster:
```

Comment:
This will not work if ~/.minikube/profiles directory does not exist.

Reply:
Right, I'm going to fix it in drenv in the next version.

Reply:
Fixed in current version.